opimodel/rules.py

```diff
@@ -2,17 +2,30 @@
 
 class Rule(object):
 
-    def __init__(self, prop_id):
+    def __init__(self, prop_id, name=None):
+        """
+
+        Args:
+            prop_id: Widget property to set
+            pv: Controlling PV
+            name (optional): Rule Name as displayed in CSS OPIEditor
+        """
         self._prop_id = prop_id
+        if name is not None:
+            self._name = name
 
 
 class BetweenRule(Rule):
 
-    def __init__(self, prop_id, pv, min_val, max_val):
+    def __init__(self, prop_id, pv, min_val, max_val,
+             gt_equals=False, lt_equals=False):
```

Comment:
This is a good way to do it, but I can't quite find the right variable names here.